tox.ini

```diff
@@ -21,6 +21,15 @@ covercmd =
       --cover-branches \
       --nocapture
 
+[testenv:py27]
+basepython =
+    python2.7
+deps =
+    {[testenv]deps}
+    grpcio >= 0.13.0
```

Comment:
tox is too magical about the sdist. The best way around this is to either do what you're doing there or do the sdist yourself:

```
[tox]
skipsdist=True

[testenv]
commands =
    pip install -e .[grpc]
    py.test

```